pulp_ansible/app/models.py

```diff
@@ -232,6 +236,64 @@ class Meta:
         unique_together = ("pubkey_fingerprint", "signed_collection")
 
 
+class AnsibleNamespaceMetadata(Content):
```

Comment:
Can you add the `AnsibleNamespace` model too? You don't have to worry about rbac or apis for this, but I think it would be good to do this now instead of later.

Also, would it be a good idea to remove `name` and instead have a fk to AnsibleNamespace?

Reply:
Ok, I'll add the model. It probably doesn't make a difference in the end, but I'll make it a fk to add the auto-gen django machinery to the models.

Reply:
I added a `namespace` foreign key, but I also left the `name` field as it is needed for the `repo_local_key` to prevent duplicate metadatas in a repo version.